src/plugins/explore/public/components/query_panel/editor_stack/top_editor/edit_toolbar/edit_toolbar.tsx

```diff
@@ -64,6 +65,8 @@ export const EditToolbar = () => {
           <EuiText size="xs">{clearText}</EuiText>
         </div>
       </EuiButtonEmpty>
+      <div className="exploreEditToolbar__verticalSeparator" />
+      <ResultsSummary />
```

Comment:
lets udate the `edit_toolbar.test.tsx`. Could you also ensure that the storybook is not busted with this change? Every component under the `query_panel` has an associated story that relies on `mock_provider.mocks.tsx`. That file might need to be updated. Try running `yarn storybook explore` and ensure its not busted